go-app-ussd_ccmdd_wc_address_update.js

```diff
@@ -273,6 +273,36 @@ go.app = (function() {
       });
     });
 
+    self.add("state_first_name", function(name) {
+      return new FreeText(name, {
+        question: $("What is your first name?"),
+        check: function(content) {
+          if (!content.match(/^\w+$/)) {
```

Comment:
Can we check if this will match certain characters like `ê`?

Reply:
Ah thanks, fixed that and removed another state, ready for another look!